web/src/infra/common/components/game/GamesList.tsx

```diff
@@ -74,8 +79,29 @@ export class GamesList extends React.Component<Props, State> {
     return this.props.showDevOnly ? 'Games In Development' : 'Games';
   }
 
+  renderGameCard(game: IGameDef) {
+    if (this.props.gamePickedCallback) {
+      return (
+        <a className={css.Card} onClick={this.onClick(game)} key={game.code}>
+          <GameCard game={game} isLink={true} />
+        </a>
+      );
+    } else {
+      return (
+        <Link href={`/play/[gameCode]`} as={`/play/${game.code}`} key={game.code}>
```

Comment:
Use passHref prop for Link component https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

Reply:
Actually not sure if this is needed

Reply:
The child isnt a custom component that wraps an <a, it is an <a...